Improve ECR creation docs

kalbir opened the pull request:
We have started using angle brackets `<` and `>` to show when a variable
needs to be filled out by the user, e.g. `<team-name>` shows that you
need to put your own team name in that slot. I've updated the example
terraform file to follow this convention.

kalbir committed:
Improve ECR creation docs

We have started using angle brackets `<` and `>` to show when a variable
needs to be filled out by the user, e.g. `<team-name>` shows that you
need to put your own team name in that slot. I've updated the example
terraform file to follow this convention.

razvan-moj-zz commented on 01-getting-started/003-ecr-setup.md:
```diff
@@ -58,14 +58,14 @@ provider "aws" {
 module "ecr-repo" {
   source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=master"
```
can you please use ref=1.0 and maybe add a note about checking available releases?

kalbir replied:
yes good point @razvan-moj

kalbir committed:
Update the docs to use specific module version

We have started to version the ECR terraform module so we need to update
the docs to show that a user should call the right version of the
module. We will need to make sure this is updated when we create a new
release of the module.
